use-product-search.md

```diff
@@ -0,0 +1,304 @@
+---
+title: Using the Product Search
+excerpt: It is possible to find products by various criteria using the `ProductSearch` class, including name, slug, taxons, channels, price ranges, sorting, and custom attributes.
+date: 2025-02-17
+tags: [models, product]
+---
+
+The `ProductSearch` class provides a powerful and flexible way to retrieve products based on various criteria. 
+
+It supports:
+
+> - Filtering by name, slug, taxon, price range, and property values
+> - Sorting options and pagination
+> - Custom attribute filtering
+
+For more details, check out the Vanilo.io [documentation](https://vanilo.io/docs/4.x/products).
```

Comment:
Move this sentence to the end of the article, here's why:

- The 3 logical chunks of a text are: 1. intro 2. matter 3. summary + next steps;
- This sentence belongs to the 3rd block;
- We do not want to redirect the reader to another page right at the beginning of the article.